octopi.js

```diff
@@ -35,4 +43,29 @@ Octopi.prototype = {
       return tbl[id];
     });
   }
+  /**
+     * Serialize the Octopi trie as string
+     *
+     * @return String
+     */
+  serialize: function(){
+    
+    var o = new Object();
+    o.uid = this.uid;
+    o.tree = this.tree;
+    o.table = this.table;
+
+    return JSON.stringify(o);
+
+  },  
+  /**
+     * Take serialized Octopi data as string and initialize the Octopi object
+     * @param json String The serialized Octopi trie
+     */
+  load: function(json){
```

Comment:
Also I think method should be added as a "class method" instead of a method on the instance:

```js
Octopi.load = function(json) {
  // ...
}
```